packages/block-editor/src/components/inserter/index.native.js

```diff
@@ -59,7 +134,42 @@ class Inserter extends Component {
 			getStylesFromColorScheme,
 		} = this.props;
 		const style = getStylesFromColorScheme( styles.addBlockButton, styles.addBlockButtonDark );
-		return renderToggle( { onToggle, isOpen, disabled, style } );
+
+		const onPress = () => {
+			this.setState( { insertionType: 'default' }, () => {
```

Comment:
`this.setState( { insertionType: 'default' }, onToggle );` is a bit shorter

Reply:
👍